app/assets/variables/_colors.scss

```diff
@@ -0,0 +1,9 @@
+$main-color: #f5c85f;
```

Comment:
color names should be related to the color and not to the context. This is because you might use the same colour for a border and a background and the idea is to be oriented on which color you are using and not to the place you are using it. 

If we wanted to create a generic website that changes only the colours it's ok to do it this way but this is not the case (rarely it is). So please change the colour names to names that correctly represent the colour